layers-samples/index.html

```diff
@@ -0,0 +1,216 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset='utf-8'>
+  <meta http-equiv='X-UA-Compatible' content='chrome=1'>
+  <meta name='viewport' content='width=device-width, initial-scale=1'>
+  <meta name='mobile-web-app-capable' content='yes'>
+  <meta name='apple-mobile-web-app-capable' content='yes'>
+
+  <meta name='twitter:card' content='summary'>
+  <meta name='twitter:title' content='WebXR Samples'>
+  <meta name='twitter:description' content='Sample WebXR pages for testing and reference'>
+
+  <link rel='icon' type='image/png' sizes='32x32' href='../favicon-32x32.png'>
+  <link rel='icon' type='image/png' sizes='96x96' href='../favicon-96x96.png'>
+
+  <link rel='stylesheet' href='../css/stylesheet.css'>
+  <link rel='stylesheet' href='../css/pygment_trac.css'>
+
+  <style>
+    article {
+      position: relative;
+      padding: 0.5em;
+      background-color: rgba(255, 255, 255, 0.90);
+      margin-bottom: 1em;
+      border-radius: 3px;
+    }
+
+    hr {
+      margin-bottom: 1em;
+    }
+
+    article h3 {
+      font-size: 1.0em;
+      margin-top: 0px;
+      margin-bottom: 0px;
+    }
+
+    article h3::before {
+      display: inline-block;
+      content: attr(data-index) ' - ';
+      font-weight: bold;
+      white-space: nowrap;
+      margin-right: 0.2em;
+    }
+
+    article h4 {
+      position: absolute;
+      right: 0.5em;
+      top: 0.5em;
+      margin-top: 0px;
+      margin-bottom: 0px;
+    }
+
+    article p {
+      margin: 0.5em;
+    }
+
+    article .links {
+      margin-left: 0.5em;
+      margin-right: 0.5em;
+    }
+
+    article a {
+      display: inline-block;
+    }
+
+    article a:not(:first-child)::before {
+      display: inline-block;
+      content: '•';
+      font-weight: bold;
+      white-space: nowrap;
+      margin-left: 0.5em;
+      margin-right: 0.5em;
+    }
+
+    .github-link {
+      font-size: 0.8em;
+    }
+
+    .wordmark img {
+      width: 70%;
+    }
+  </style>
+
+  <!--[if lt IE 9]>
+    <script src='https://html5shiv.googlecode.com/svn/trunk/html5.js'></script>
+    <![endif]-->
+  <title>WebXR - Layers Samples</title>
+</head>
+
+<body>
+
+  <div class='container' id='container'>
+    <header class='header'>
+      <div id='nav'>
+        <a href='../'>Samples</a>
+        <a href='./' class='selected'>Layers Samples</a>
```

Comment:
```suggestion
        <a href='./' class='selected'>Layers Samples</a>
        <a href='proposals/'>Proposals</a>
```

Just for consistency